website/docs/dump-load-restart.md

```diff
@@ -22,6 +22,10 @@ $ starknet-devnet --dump-on block --dump-path <PATH>
 $ starknet-devnet --dump-on exit --dump-path <DEFAULT_PATH>
 ```
 
+```
+POST /dump
+```
+
 ```
 POST /dump
 {
```

Comment:
Would it be clearer if these two snippets were merged into one? Perhaps with a comment indicating that the whole body is optional, as well as the path parameter?

Reply:
[ad38af3](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/543/commits/ad38af3fc7fb121a9ae5ab07bb484ec5b790c8dc)